src/main.rs

```diff
@@ -1083,6 +1088,7 @@ impl Info {
             Language::Vue => vec![Color::BrightGreen, Color::Blue],
             Language::Perl => vec![Color::BrightBlue],
             Language::Php => vec![Color::BrightWhite],
+            Language::XML => vec![Color::Yellow],
```

Comment:
You seem to be using two colors in the ascii logo but you're only referencing a single value in the vector, it still works though